### DISCUSSION TITLE
Will photoprism support "share photo url to someone"?

### DISCUSSION THREAD

**Discussion by travisbikkle** [photoprism/photoprism#2114] [Ideas]
I wanted to share one video with my friend who already has admin permission to photoprism,
I hope she open that url, login and then able to see that video.
But i only got that 'share to webdav' window.
Is this feature valuable?

**Comment by Alestrix** [comment]
You could add that photo/video to a new album and share the link to that album. Not the same, but maybe a viable workaround.
If you share the `/albums/...` path, your friend will have to log in. If you share the album as `/s/...` path, login will not be needed.
